luigi/contrib/bigquery.py

```diff
@@ -371,9 +391,11 @@ def copy(self,
 
 
 class BigQueryTarget(luigi.target.Target):
-    def __init__(self, project_id, dataset_id, table_id, client=None, location=None):
+    def __init__(self, project_id, dataset_id, table_id, client=None, location=None, enable_chunking=False, chunk_size_gb=1000):
```

Comment:
Also please document what reasons would people possibly have to not enable chunking. Otherwise why even have it toggleable if chunking works fine for all use cases?

Isn't setting chunk_size_gb to "infinity" the same as disabling chunking? Then also perhaps the `enable_chunking` parameter is superfluous.